src/Analysis/Engine/Impl/Values/BuiltinFunctionInfo.cs

```diff
@@ -121,26 +122,10 @@ public override IAnalysisSet Call(Node node, AnalysisUnit unit, IAnalysisSet[] a
             }
         }
 
-        public override IEnumerable<OverloadResult> Overloads {
-            get {
-                if (_overloads == null) {
-                    var overloads = Function.Overloads;
-                    var result = new OverloadResult[overloads.Count];
-                    for (int i = 0; i < result.Length; i++) {
-                        result[i] = new BuiltinFunctionOverloadResult(ProjectState, Function.Name, overloads[i], 0, () => Description);
-                    }
-                    _overloads = new ReadOnlyCollection<OverloadResult>(result);
-                }
-                return _overloads;
-            }
-        }
+        public override IEnumerable<OverloadResult> Overloads
+            => GetFunctionOverloads().Select(o => new BuiltinFunctionOverloadResult(ProjectState, Function.Name, o, 0, () => Description));
 
-        public override string Documentation {
-            get {
-                _doc = _doc ?? Utils.StripDocumentation(Function.Documentation);
-                return _doc;
-            }
-        }
+        public override string Documentation => _doc = _doc ?? (_doc = Utils.StripDocumentation(Function.Documentation));
```

Comment:
`public override string Documentation => _doc ?? (_doc = Utils.StripDocumentation(Function.Documentation));` should be enough